hack/create-git-release.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -euo pipefail
+
+REPODIR="$(dirname "$0")/.."
+
+if [[ ! -f "${REPODIR}/release" ]]; then
+  echo "Release name file (release) does not exist. Nothing to do"
+fi
+if [[ ! -f "${REPODIR}/release-date" ]]; then
+  echo "Release date file (release-date) does not exist. Nothing to do"
+fi
+
+RELEASE="$(cat "${REPODIR}/release")"
+RELEASE_DATE="$(cat "${REPODIR}/release-date")"
+RELEASE_TAG="v${RELEASE}-${RELEASE_DATE}"
```

Comment:
We are currently using `+` not `-`.  Although we are using `-` for the image tags.

<img width="1267" alt="image" src="https://user-images.githubusercontent.com/17885615/155992639-fcdf1e6b-e0c9-4481-be1d-94019f381022.png">

Reply:
@rtheis thank you for catching that. I will fix

Reply:
Fixed